_posts/2025-01-30-release-0_10_0.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Kroxylicious release 0.10.0"
+date:   2025-01-30 00:00:00 +0000
+author: "Grace Grimwood"
+author_url: "https://github.com/gracegrimwood"
+# noinspection YAMLSchemaValidation
+categories: blog kroxylicious releases
+tags: [ "releases", "kroxylicious" ]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.10.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.10.0). See the [Changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#0100) for a list of changes and summary of Deprecations, Changes and Removals.
+
+### Fortanix KMS for Record Encryption
+
+This release introduces initial support for using [Fortanix Data Security Manager](https://www.fortanix.com/platform/data-security-manager) as a Key Management System provider with the Kroxylicious Record Encryption filter ([#1672](https://github.com/kroxylicious/kroxylicious/pull/1672)).
+
+### Improvements and tweaks to Record Encryption
+
+We've introduced some changes in this release to Encryption DEK invalidation logic to improve coordination between threads ([#1643](https://github.com/kroxylicious/kroxylicious/pull/1643)), and added some restrictions to TLS protocols and cipher suites used by Key Management System implementations to improve the overall security of Kroxylicious ([#1743](https://github.com/kroxylicious/kroxylicious/pull/1743)).
```

Comment:
The cipher suite and protocol restrictions also apply to downstream connections not just the KMS

Reply:
upstream and downstream kafka connections.

Reply:
`added some restrictions` sounds a bit like we've changed some default behaviour. Is this better described as "enabling users to control which TLS protocols and cipher suites are enabled for ..."